src/components/categoryImages/CategoryImage.tsx

```diff
@@ -2,12 +2,12 @@ import React from 'react'
 import {ImageParent, Img, Button} from './styleCategroyImage'
 import Frame from '../../assets/frame2.png'
 
-const CategoryImage = () => {
+const CategoryImage = ({img , key, title  } : { img: string; key:number ; title: string }) => {
```

Comment:
Consider using a code formatter like prettier, like here you have some unncessary spaces can be catched by the formatter